ANGULAR/Task_2/User-Management-App/src/app/shared/user-details/user-details.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="user-details-div" *ngIf="user">
+    <div class="user-name">{{user.firstName}} {{user.lastName}}</div>
+    <div [ngStyle]="{'color': user.isDeleted?'red':'green' }">{{user.isDeleted?'Deactive':'Active'}}</div>
```

Comment:
For better readability move the conditions to the component as a function and return the values from there